ingestors/arena-ingestor/src/main/kotlin/no/nav/amt/tiltak/ingestors/arena/processors/DeltakerStatusConverter.kt

```diff
@@ -0,0 +1,67 @@
+package no.nav.amt.tiltak.ingestors.arena.processors
+
+import io.micrometer.core.instrument.MeterRegistry
+import io.micrometer.core.instrument.Tag
+import no.nav.amt.tiltak.core.domain.tiltak.Deltaker
+import org.springframework.stereotype.Service
+import java.time.LocalDate
+
+
+private typealias ConversionStrategy = (DateRange) -> Deltaker.Status
+
+@Service
+internal class DeltakerStatusConverter(
+	private val meterRegistry: MeterRegistry
+) {
+
+	private val alleStatuser: Map<String, ConversionStrategy> = mapOf(
+		"AKTUELL" to { Deltaker.Status.IKKE_AKTUELL }, // Aktuell
+		"AVSLAG" to { Deltaker.Status.IKKE_AKTUELL }, // Fått avslag
+		"DELAVB" to { if (it.startDatoPassert()) Deltaker.Status.HAR_SLUTTET else Deltaker.Status.IKKE_AKTUELL }, // Deltakelse avbrutt
```

Comment:
Er det egentlig om startdato har passert(i dag) som er relevant her for hva statusen skal/skulle vært? Det vil vel i praksis si at alle som har avbrutt deltakelse vil "ha sluttet" så lenge man venter til etter kurset skulle ha funnet sted. Om man skal bry seg om det som allerede ligger på kafka da, eller om man aksepterer at de gamle får potensielt feil status. Kan evt bruke feltet dato for siste statusendring, tror det kan være mer riktig.

Reply:
Snakket med Linn om det, og hørtes ut som det var sånn det var tenkt ja

Reply:
For å være presis, så er det tegnet inn i whimsical, men trellokortet for det står fortsatt i analyse. Tolket det som det var sannsynlig at det blir sånn som i whimsical. Implementerte det derfor på den måten - siden jeg tenkte det er enklere å se eventuelle feil ta litt i en løsning

Reply:
Ok, det er ikke så lett å forstå det som er tegnet i whimsical. Men uansett, hvis det er en annen oppgave på det så får man evt endre det der